src/assets/translations/nl.json

```diff
@@ -25,29 +25,29 @@
         "protectedContent": "Deze beschermde video kan niet worden afgespeeld.",
         "technicalError": "Deze video kan niet worden afgespeeld door een technisch probleem."
     },
-    "exitFullscreen": "Volledig Scherm Verlaten",
+    "exitFullscreen": "Volledig Scherm verlaten",
```

Comment:
I would not capitalize Scherm here, just like in another example where "Volledig scherm" is used further below and is not capitalized. I don't think this is a strict rule, but it's definitely more consistent.

Reply:
I kinda chose it because of the meaning of 'Volledig Scherm' being a mode on the JW player, but will change it.